src/Components/Drink.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import '../Styles/Drink.css'
+
+const Drink = () => {
+
+    const currentDrink = useSelector(state => {
+        return state.favoriteDrinks.currentDrink
+    })
+
+    const getIngredients = (keys) => {
+        return keys.reduce((ingredients, each) => {
+            if (each.includes('strIngredient') && currentDrink[each]) {
+                ingredients.push(currentDrink[each])
+            }
+            return ingredients
+        }, [])
+    }
+
+    const getMeasures = (keys) => {
+        return keys.reduce((measures, each) => {
+            if (each.includes('strMeasure') && currentDrink[each]) {
+                measures.push(currentDrink[each])
+            }
+            return measures
+        }, [])
+    }
+
+    const makeTheSameLength = (ingredients, measures) => {
+        if (ingredients.length === measures.length) {
+            return {ingredients, measures}
+        } else if (ingredients.length > measures.length) {
+            let difference = ingredients.length - measures.length
+            for (let i = 0; i < difference; i++) {
+                measures.push('')
+            }
+            return {ingredients, measures}
+        } else {
+            return {ingredients: [], measures: []}
+        }
+    }
+
+    const createIngredientAmount = (currentDrink) => {
```

Comment:
Joan - such a good job on creating these functions. The logic seems really solid to me. I'm not sure of ways to clean this up or shorten this code at this exact moment but I'm sure we'll all be thinking of this as we finish up our work.